### PR TITLE
Update supporting platform

### PR DESCRIPTION
**Pull request by peifeng-unity**
## Proposed change(s)
- [x] Update version to 2020.3.11f1
- [x] add UWP to supporting platform because there are a few open issues related to the UWP platform

### Useful links (GitHub issues, JIRA tickets, forum threads, etc.)

https://github.com/Unity-Technologies/Unity-Robotics-Hub/issues/254

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Documentation update
- [x] Other (please describe)

## Checklist
- [x] Ensured this PR is up-to-date with the `dev` branch
- [x] Created this PR to target the `dev` branch
- [ ] Followed the style guidelines as described in the [Contribution Guidelines](https://github.com/Unity-Technologies/ROS-TCP-Connector/blob/main/CONTRIBUTING.md)
- [ ] Added tests that prove my fix is effective or that my feature works
- [x] Updated the [Changelog](https://github.com/Unity-Technologies/ROS-TCP-Connector/blob/dev/com.unity.robotics.ros-tcp-connector/CHANGELOG.md) and described changes in the [Unreleased section](https://github.com/Unity-Technologies/ROS-TCP-Connector/blob/dev/com.unity.robotics.ros-tcp-connector/CHANGELOG.md#unreleased)
- [ ] Updated the documentation as appropriate

## Other comments

### CODE REVIEW

File: TestRosTcpConnector/ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.2.0b9
-m_EditorVersionWithRevision: 2020.2.0b9 (ef2968fa77ae)
+m_EditorVersion: 2020.3.11f1
```
Comment (sdiao) [inline TestRosTcpConnector/ProjectSettings/ProjectVersion.txt:1]: is the editor version upgrade inentional?

**Review by hyounesy — COMMENTED** [review]
Are all changes to all the files other than the Unity.Robotics.ROSTCPConnector.asmdef really necessary for the UWP support?

File: TestRosTcpConnector/ProjectSettings/ProjectVersion.txt (hunk 1 above)
Comment (peifeng-unity) [reply TestRosTcpConnector/ProjectSettings/ProjectVersion.txt:1]: Yes, I thought we were saying to upgrade the Unity versions to LTS. The [Pick-and-Place ](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/dev/tutorials/pick_and_place/PickAndPlaceProject/ProjectSettings/ProjectVersion.txt)and [URDF-Importer](https://github.com/Unity-Technologies/URDF-Importer/blob/dev/TestUrdfImporter/ProjectSettings/ProjectVersion.txt) have been updated.

**Comment by peifeng-unity** [comment]
> Are all changes to all the files other than the Unity.Robotics.ROSTCPConnector.asmdef really necessary for the UWP support?

Yeah, all the other changes are related to the version upgrade from 2020.2.0b9 to 2020.3.11f1. I thought we were saying to upgrade the Unity versions to LTS. The [Pick-and-Place ](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/dev/tutorials/pick_and_place/PickAndPlaceProject/ProjectSettings/ProjectVersion.txt)and [URDF-Importer](https://github.com/Unity-Technologies/URDF-Importer/blob/dev/TestUrdfImporter/ProjectSettings/ProjectVersion.txt) have been updated.

**Review by mrpropellers — APPROVED** [review]
Fine with this assuming no objections from the other reviewers!

**Review by sdiao — APPROVED** [review]

**Review by hyounesy — APPROVED** [review]
Tested by creating a UWP build and verified that:
- [x] There is a build error when creating a UWP build before the fix
- [x] The build error is fixed after the fix.

**Pull request merged**